src/manifest.ts

```diff
@@ -229,6 +234,7 @@ export class Manifest {
     this.bootstrapSha = manifestOptions?.bootstrapSha;
     this.lastReleaseSha = manifestOptions?.lastReleaseSha;
     this.draft = manifestOptions?.draft;
+    this.prerelease = manifestOptions?.prerelease;
```

Comment:
I don't think we need to capture this one because it's configured on the per-component config (the default is pulled from the root of the manifest config on new line 1078)

Reply:
it was a bit confusing for me and for this part, I was trying to follow the logic of `draft`. In that case, should the line 236 also be removed?

Reply:
done